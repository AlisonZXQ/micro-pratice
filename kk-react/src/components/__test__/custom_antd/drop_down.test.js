import React from 'react';
import { shallow } from 'enzyme';
import mountTest from '@tests/shared/mountTest';
import DropDown from '../../CustomAntd/drop_down';

describe('drop_down', () => {
  mountTest(DropDown);

  it('should snapshot correctly', () => {
    const shallowWrapper = shallow(<DropDown />);
    expect(shallowWrapper).toMatchSnapshot();
  });
})

