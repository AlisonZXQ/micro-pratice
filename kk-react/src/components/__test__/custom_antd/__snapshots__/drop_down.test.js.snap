// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`drop_down should snapshot correctly 1`] = `
<span
  onClick={[Function]}
>
  <Dropdown
    mouseEnterDelay={0.15}
    mouseLeaveDelay={0.1}
    onClick={[Function]}
    placement="bottomLeft"
    trigger={
      Array [
        "click",
      ]
    }
  >
    <span
      className="caozuo"
    >
      更多
      <MyIcon
        className="icon"
        type="icon-gengduozhankai"
      />
    </span>
  </Dropdown>
</span>
`;
